fsfreeze.go

```diff
@@ -0,0 +1,150 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"syscall"
+)
+
+// from linux/fs.h
+const FIFREEZE = 0xC0045877
+const FITHAW = 0xC0045878
+
+const guestFsfreezeStatusThawed = "thawed"
+const guestFsfreezeStatusFrozen = "frozen"
```

Comment:
I assume these are constants required by the spec.

Maybe comment like `// spec constants`